### DISCUSSION TITLE
How & if at all the application able to lock a study being annotated by other users?

### DISCUSSION THREAD

**Discussion by hiq-ayeddula** [Project-MONAI/MONAILabel#1643] [Q&A]
In case of multiple annotators annotating the same dataset, how & if at all the application able to lock a study being annotated by other users

**Comment by diazandr3s** [comment]
Hi @hiq-ayeddula,

This is a great question. 

If multiple annotators work with the same dataset, MONAI Label sends an unlabeled volume to each user depending on the strategy selected (random, first, etc)

Currently, there is no mechanism to ensure each user doesn't get the same volume - there is a risk of getting the same volume.

However, it shouldn't be difficult to freeze an unlabeled volume for some time so others fetch a different one. It can be done in the datastore: https://github.com/Project-MONAI/MONAILabel/tree/main/monailabel/datastore

Hope this makes sense,
